src/side-navigation/__tests__/nav-long.scenario.js

```diff
@@ -0,0 +1,86 @@
+/*
+Copyright (c) 2018-2019 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+
+import * as React from 'react';
+
+import {Navigation} from '../index.js';
+
+export const name = 'side-navigation-long';
+
+function createItems(floor) {
+  const nav = [];
+  for (let i = 0; i < 300; i++) {
+    nav.push({
+      title: `item ${i + floor + 1}`,
+      itemId: String(i),
+    });
+  }
+  return nav;
+}
+
+function areEqualShallow(a, b) {
+  if (!a || !b) return false;
+  if (typeof a !== 'object' || typeof a !== 'object') return false;
```

Comment:
I think there's a typo here and the second condition should be `typeof b`

Reply:
thank you!

```suggestion
  if (typeof a !== 'object' || typeof b !== 'object') return false;
```